webapp/src/main/java/com/box/l10n/mojito/service/security/user/session/CustomCreateSessionAttributeInsertQueryConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package com.box.l10n.mojito.service.security.user.session;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.session.jdbc.JdbcIndexedSessionRepository;
+import org.springframework.session.jdbc.config.annotation.web.http.JdbcHttpSessionConfiguration;
+import org.springframework.util.StringUtils;
+
+
+/**
+ * Workaround for issue <a href="https://github.com/spring-projects/spring-session/issues/1213">Concurrent requests adding the same session attribute result in duplicate key constraint violation</a>
+ * to change the Create Session Attribute query to use an MySQL UPDATE command to update the ATTRIBUTE_BYTES
+ * of the existing row if a duplicate key is encountered.
+ *
+ * <b>NOTE:</b>This custom insert query should only be used with a MySQL database.
```

Comment:
`only be used with a MySQL database`, can this be made explicit with a `@Conditional`?

Reply:
I've pushed an update containing a custom condition (MySQLDatabaseCondition) that checks the prefix of the `spring.datasource.url` property, if the prefix matches "jdbc:mysql:" the condition passes